tests/integration/component_provider/nodejs/component-provider-host/yaml/Pulumi.yaml

```diff
@@ -1,8 +1,6 @@
 name: nodejs-component-provider-yaml
 description: Using a component provider written in Node.js from YAML
 runtime: yaml
-packages:
```

Comment:
Let `pulumi package add ...` add the package. This isn't even the right name for the provider.